sale_product_variant_attribute_tax/models/sale_order.py

```diff
@@ -10,7 +10,7 @@ class SaleOrderLine(models.Model):
     def _onchange_product_attribute_ids_configurator(self):
         res = super()._onchange_product_attribute_ids_configurator()
         # if there's a product set, taxes on that product have priority
-        if self.product_id or not self.product_tmpl_id:
+        if not self.product_tmpl_id:
```

Comment:
Why this change is necessary?

Reply:
If product is set we need to recalculate

Reply:
IMO the above behavior is correct or we will always define the taxes of the attribute omitting completely those of the product template, the above behavior causes it to be defined only when there is no `product_id` defined yet (the template has several attributes and only some of them have been defined but not all of them yet).

Reply:
As it was, it was working correctly, so better to not change it unless something have changed across the versions.

Reply:
I confirm you shouldn't modify this. Please leave it as it was.

Reply:
Please check without this condition if not self.product_tmpl_id, the module is not working properly.